python/ray/tests/BUILD

```diff
@@ -174,11 +175,12 @@ py_test_module_list(
     "test_advanced.py",
     "test_basic.py",
     "test_basic_2.py",
+    "test_basic_3.py",
   ],
   size = "medium",
   extra_srcs = SRCS,
   name_suffix = "_client_mode",
-  # TODO(barakmich): py_test will support env in Bazel 4.0.0... 
+  # TODO(barakmich): py_test will support env in Bazel 4.0.0...
```

Comment:
lol meanwhile no other part of the ray build works on bazel 4.0 😂

Reply:
lol